hera_cal/lstbin.py

```diff
@@ -544,17 +519,18 @@ def lst_bin_files(data_files, filetype='uvh5', dlst=None, verbose=True, ntimes_p
     Parameters:
     -----------
     data_files : type=list of lists: nested set of lists, with each nested list containing
-                 paths to files from a particular night. These files should be sorted
-                 by ascending Julian Date. Frequency axis of each file must be identical.
+        paths to files from a particular night. These files should be sorted
+        by ascending Julian Date. Frequency axis of each file must be identical.
     filetype : str, options=['uvh5', 'miriad']
     dlst : type=float, LST bin width. If None, will get this from the first file in data_files.
     lst_start : type=float, starting LST for binner as it sweeps from lst_start to lst_start + 2pi.
     fixed_lst_start : type=bool, if True, LST grid starts at lst_start, regardless of LST of first data
         record. Otherwise, LST grid starts at LST of first data record.
     ntimes_per_file : type=int, number of LST bins in a single output file
-    file_ext : type=str, extension to "zen." for output files. This must have three
-               formatting placeholders, first for polarization(s), second for type of file
-               Ex. ["LST", "STD", "NUM"] and third for starting LST bin of file.
+    file_ext : type=str, extension to "zen." for output files. This must have at least a ".{type}." field
```

Comment:
Shouldn't the default be 4pol files? Why include {pol}?

Reply:
to be backwards compatible.

Reply:
I'd change the default kwarg, but mention in the docstring how to be backwards compatible.